test/integration/docker_test.go

```diff
@@ -158,3 +164,114 @@ func TestForceSystemdEnv(t *testing.T) {
 		validateContainerdSystemd(ctx, t, profile)
 	}
 }
+
+// TestDockerEnvContainerd makes sure that minikube docker-env command works when the runtime is containerd
+func TestDockerEnvContainerd(t *testing.T) {
+	t.Log("running with", ContainerRuntime(), DockerDriver(), runtime.GOOS, runtime.GOARCH)
+	if ContainerRuntime() != constants.Containerd || !DockerDriver() || runtime.GOOS != "linux" || runtime.GOARCH != "amd64" {
```

Comment:
```suggestion
	if ContainerRuntime() != constants.Containerd || !DockerDriver() || runtime.GOOS != "linux" {
```
This should work on arm64 as well